test/service/test_jobs.py

```diff
@@ -366,13 +368,20 @@ def test_throws_when_scene_is_missing_required_bands(self):
             jobs.create(API_KEY, 'test-user-id', 'test-scene-id', 'test-algo-id', 'test-name')
 
     def test_throws_when_catalog_is_unreachable(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.